python/ray/experimental/serve/__init__.py

```diff
@@ -0,0 +1,12 @@
+import sys
+if sys.version_info < (3, 0):
+    raise ImportError("Ray Serve is Python 3 Only.")
```

Comment:
```suggestion
    raise ImportError("serve is Python 3 only.")
```

Reply:
Ping on this. I'm fine with either:
`Ray serve is Python 3 only.`
or
`serve is Python 3 only.`